src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -87,7 +89,32 @@
 #include "jfr/jfr.hpp"
 #endif
 
-SymbolPropertyTable*   SystemDictionary::_invoke_method_table = NULL;
+class InvokeMethodKey : public StackObj {
```

Comment:
It is odd to declare as `StackObj`, should be value object instead.

Reply:
I used the key as a StackObj in the code, it is implicitly copied into the hashtable, so technically it is a value object.  But since I explicitly used it as a StackObj I would like to keep it this way.